src/ui/components/DevTools.js

```diff
@@ -127,6 +127,10 @@ function DevTools({
     return null;
   }
 
+  if (document.title != data.recordings[0].title) {
+    document.title = data.recordings[0].title;
+  }
```

Comment:
We should put this into a `useEffect`. It's not necessary, but I think it's a bit more proper